src/lib/operators/join_hash.cpp

```diff
@@ -214,8 +263,15 @@ class JoinHash::JoinHashImpl : public AbstractJoinOperatorImpl {
           ChunkOffset offset = 0;
           for (auto&& elem : materialized_chunk) {
             if (elem.first.chunk_offset != INVALID_CHUNK_OFFSET) {
-              output[row_id] =
-                  PartitionedElement<T>{RowID{chunk_id, offset}, murmur2<T>(elem.second, seed), elem.second};
+              unsigned int hashed_value;
```

Comment:
I know, `murmur2` returns `unsigned int`s, but could we change this to a fixed width integer type?